independent-projects/tools/devtools-common/src/main/java/io/quarkus/platform/catalog/processor/CatalogProcessor.java

```diff
@@ -8,12 +8,18 @@
 
 public class CatalogProcessor {
     private static final String CODESTART_ARTIFACTS = "codestarts-artifacts";
-    private static final String UNCATEGORIZED_ID = "uncategorized";
-    private static final String UNCATEGORIZED_NAME = "Uncategorized";
-    private static final String UNCATEGORIZED_DESCRIPTION = "The category is not defined for those extensions.";
+    private static final Category UNCATEGORIZED_CATEGORY;
 
     private final ExtensionCatalog catalog;
 
+    static {
+        JsonCategory uncategorized = new JsonCategory();
+        uncategorized.setId("uncategorized");
+        uncategorized.setName("Uncategorized");
+        uncategorized.setDescription("The category is not defined for those extensions.");
+        UNCATEGORIZED_CATEGORY = uncategorized;
+    }
```

Comment:
Add a constructor to `JsonCategory` which takes these parameters instead of using a static block here

Reply:
Or even better, create a `JsonCategory.UNCATEGORIZED` attribute there ;)

Reply:
`JsonCategory` is really low level (part of the api), I prefer not to touch it